packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -238,6 +238,14 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         }
     }
 
+    private _getChannelName() {
```

Comment:
should this be channel specific url instead of channel name?

Reply:
I think it would be more understandable if we left it as channle_url instead of name, assigned channel_name due to the ticket description. What are your thoughts?

Reply:
Since it is not an actual channel name, and we have code like this `channel_name.split('://')[1];`, could you please rename it?

Reply:
What do you think of channel_link?

Reply:
Perhaps `channel_url`?